### PR TITLE
Update PPP

### PR DESCRIPTION
**Pull request by leezhixuan**
Updated DG and UG to replace instances of 'findAny' with 'findOr' as well.
Contributes to #171

### CODE REVIEW

**Review by Justinhoejj — APPROVED** [review]

File: docs/team/leezhixuan.md [hunk 1]
```diff
@@ -5,25 +5,58 @@ title: Lee Zhi Xuan's Project Portfolio Page
...
 * **Project management**:
-
+  * Set up Github Pages
+  * Created and assigned issues for milestone `v1.3`.
+  <br></br>
+    
 * **Enhancements to existing features**:
-
+  * Redefine equality amongst contacts.
+    * Previously, equality between two contacts hinged on the fact that they had the same name. Since people can have friends with the same name, I changed CONNECTIONS to check for phone numbers instead to raise possible duplicate contacts.
```
[the remark this answers is not in the document]
Comment (leezhixuan) [reply docs/team/leezhixuan.md]: done

File: docs/team/leezhixuan.md [hunk 2]
```diff
@@ -5,25 +5,60 @@ title: Lee Zhi Xuan's Project Portfolio Page
 
 ### Project: CONNECTIONS
 
-CONNECTIONS is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+CONNECTIONS is a desktop address book application adapted from AB3. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java and has about 10 KLOC.
```
Comment (yucheng11122017) [inline docs/team/leezhixuan.md:8]: WHY YOU CAN CHANGE THIS I CANNOT

File: docs/team/leezhixuan.md [hunk 3]
```diff
@@ -5,25 +5,60 @@ title: Lee Zhi Xuan's Project Portfolio Page
...
 
 Given below are my contributions to the project.
 
-* **New Feature**:
-
+* **New Feature**: FindOr
```
Comment (yucheng11122017) [inline docs/team/leezhixuan.md:12]: Still findOr bro..

File: docs/team/leezhixuan.md [hunk 4]
```diff
@@ -5,25 +5,60 @@ title: Lee Zhi Xuan's Project Portfolio Page
...
+  * What it does: 
+    * FindOr is a command that allows users to apply an 'OR' filter with the search terms they have provided.
```
Comment (yucheng11122017) [inline docs/team/leezhixuan.md:14]: Same as above :)

File: docs/team/leezhixuan.md [hunk 5]
```diff
@@ -5,25 +5,60 @@ title: Lee Zhi Xuan's Project Portfolio Page
...
+  * Justification:
+    * "OR" search retrieves records that include either terms searched for. Using OR broadens a search and thus returns more results. 
+      This way, users will be able to filter contacts with greater flexibility.
+      <br></br>
```
Comment (yucheng11122017) [inline docs/team/leezhixuan.md:18]: Why is there this br here?

File: docs/team/leezhixuan.md [hunk 6]
```diff
@@ -5,25 +5,60 @@ title: Lee Zhi Xuan's Project Portfolio Page
...
+  <br></br>
+    
 * **Enhancements to existing features**:
-
+  * Redefine equality amongst contacts.
+    * Previously, equality between two contacts hinged on the fact that they had the same name. Since people 
+      can have friends with the same name but unique phone numbers, I changed CONNECTIONS to check for phone 
+      numbers instead to raise possible duplicate contacts.
+  * Revamp Find 
+    * Find ('AND'-search) and FindOr ('OR'-search) can be applied names and/or tags.
```
Comment (yucheng11122017) [inline docs/team/leezhixuan.md:40]: findAny

**Review by yucheng11122017 — APPROVED** [review]
LGTM except for some minor stuff! :)

File: docs/team/leezhixuan.md (hunk 5 above)
Comment (leezhixuan) [reply docs/team/leezhixuan.md:18]: line break

File: docs/team/leezhixuan.md (hunk 2 above)
Comment (leezhixuan) [reply docs/team/leezhixuan.md:8]: shit.......

File: docs/team/leezhixuan.md (hunk 3 above)
Comment (leezhixuan) [reply docs/team/leezhixuan.md:12]: sigh..... my bad....

**Pull request merged**